client/main.py

```diff
@@ -1,13 +1,20 @@
 import time
 import pygame
 import sys
+
+from pygame_menu import Menu
 import game.visuals as visuals
 import game.cache as local_cache
 import connection as conn
+import game.menu as Menu
 
 
 visuals.initialize_game()
 
+# Menu loop
+def menu():
```

Comment:
Menu needs to return `True`  while the user has not opted to play. It needs to return `False` when the user decides to join. Take a look at line 94 to see how the return value of this function will allow the `play` function to start.